docs/modules/ROOT/pages/advanced-scripting.adoc

```diff
@@ -537,73 +537,6 @@ Fun 1
 
 == Output
 
-=== Type of CLC Output
-
-CLC supports two kinds of output: structured and "unnecessary".
```

Comment:
why the quotes around unnecessary? They don't provide any extra information. Suggest:

CLC supports both structured and unnecessary types of output.

Reply:
@rebekah-lawrence Thanks for the review!
But I think you've reviewed an older version of the guide.
I've sent a few other commits right after asking review from you.
And your comments in this PR are all for the removed text.
Sorry for that.

Reply:
@yuce you are quite right. Obviously, I should have had another cup of tea before starting *face palm*.